timesheet_task/__openerp__.py

```diff
@@ -23,7 +23,7 @@
  'author': "Camptocamp,Odoo Community Association (OCA)",
  'maintainer': 'Camptocamp, Acsone SA/NV',
  'category': 'Human Resources',
- 'depends': ['project', 'hr_timesheet_invoice'],
+ 'depends': ['project', 'analytic_user_function'],
```

Comment:
Because on_change_user_id has its signature changed in analytic_user_function and breaks this module when it is installed. Check related issue in PR description.